core/src/main/scala/org/apache/spark/h2o/backends/SharedBackendUtils.scala

```diff
@@ -154,6 +141,11 @@ private[backends] trait SharedBackendUtils extends Logging with Serializable {
       .add("-jks", conf.jks)
       .add("-jks_pass", conf.jksPass)
       .add("-jks_alias", conf.jksAlias)
+      .addIf("-hash_login", conf.hashLogin)
```

Comment:
I actually meant moving all the properties from `getLoginArgs` inside the method `getH2OCommonArgs` to ensure it's on both client & worker nodes

Reply:
ah, wrong method. I originally wanted to move it to `getH2OCommonArgs`. My bad, thanks!

Reply:
👍